scdl/scdl.py

```diff
@@ -552,10 +553,36 @@ def download_track(track, playlist_name=None, playlist_file=None):
     filetime = int(time.mktime(timestamp.timetuple()))
     try_utime(filename, filetime)
 
+    #Download track cover art
+    if arguments['--write-cover'] and track['artwork_url']:
+        logger.info('Downloading track art...')
+        download_track_art(track)
+        
+
     logger.info('{0} Downloaded.\n'.format(filename))
     record_download_archive(track)
 
+def download_track_art(track):
```

Comment:
https://github.com/flyingrub/scdl/blob/54e2ac0e48dcc0bcf7e9c2415787a7ad69013ffb/scdl/scdl.py#L672, We already download the artwork here, you could just check for the `arguments['--write-cover']` here, also I think we can just keep the title as filename without the ` TRACK ART` string.